cmdstanpy/model.py

```diff
@@ -538,19 +612,16 @@ def run_generated_quantities(
                 data=_data,
                 seed=seed,
                 output_basename=gq_csv_basename,
-                method_args=generate_quantities_args
+                method_args=generate_quantities_args,
```

Comment:
don't need final comma  (nit-picky)

Reply:
I think that comes from `black`

Reply:
agreed - so it's fine to leave as is, and that's why I went ahead and merged.
I'd like to do another release to PyPi in order get parallelism back.

Reply:
New release sounds good